docker/dockerfile.tf

```diff
@@ -15,7 +15,7 @@ FROM ${BASE_IMAGE} as base
 COPY --chown=1000:1000 --from=triton /opt/tritonserver/backends/tensorflow2 backends/tensorflow2/
 
 # Tensorflow dependencies (only)
-RUN pip install tensorflow-gpu \
+RUN pip install tensorflow-gpu==2.9.2 \
```

Comment:
Is this pinning us to 2.9.2?  Is there a way to reflect this as a range?  I'm guessing not, that's more for dependencies.

Can you add a comment on the commit about why we're pinning as well.